### PR TITLE
Fix meeco.kr

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/91294

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -4362,6 +4362,8 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/91294
+meeco.kr#?#div[style^="padding:"] > p > img[src^="//img.meeco.kr/files/attach/images/"][alt*="배너"]:upward(.xe_content)
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:4366]: Please move it to the Mobile filter - css_extended.txt.
Also change a bit rule `meeco.kr#?#div[style^="padding:"] > p img[src^="//img.meeco.kr/files/attach/images/"][alt*="배너"]:upward(.xe_content)`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (piquark6046) [reply EnglishFilter/sections/foreign.txt]: Done. https://github.com/AdguardTeam/AdguardFilters/pull/91309/commits/294089f6e35852a7e30cc388e455ec39b2b1512a

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: It's a specific_web.txt.
<details><summary>Screenshot:</summary>

image

</details><br/>

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: I moved it to the right section.

**Pull request merged**